bin/ch-test

```diff
@@ -869,7 +874,7 @@ fi
 
 printf "%-21s %s" 'phase:' "$phase"
 if [[ $phase = one-file ]]; then
-    printf ': %s' "$one_file"
+    printf ': %s (%s)' "$one_file" "$ch_one_test"
```

Comment:
We are OK with empty parens if the one test variable is empty?

Reply:
I guessed that was OK since the audience is debugging and we have a caveat to that effect in the docs.